sdk/search/azure-search-documents/azure/search/documents/__init__.py

```diff
@@ -55,32 +55,30 @@
     ConditionalSkill,
     CorsOptions,
     CustomAnalyzer,
-    DataSource,
     DataSourceCredentials,
-    DataContainer,
     DictionaryDecompounderTokenFilter,
     DistanceScoringFunction,
     DistanceScoringParameters,
     EdgeNGramTokenFilter,
     EdgeNGramTokenizer,
     ElisionTokenFilter,
-    EncryptionKey,
     EntityRecognitionSkill,
-    Field,
     FreshnessScoringFunction,
     FreshnessScoringParameters,
     GetIndexStatisticsResult,
     ImageAnalysisSkill,
-    Index,
-    Indexer,
     InputFieldMappingEntry,
     KeepTokenFilter,
     KeyPhraseExtractionSkill,
     KeywordMarkerTokenFilter,
     KeywordTokenizer,
     LanguageDetectionSkill,
     LengthTokenFilter,
+    LexicalAnalyzer,
+    LexicalTokenizer,
     LimitTokenFilter,
+    LuceneStandardAnalyzer,
+    LuceneStandardTokenizer,
```

Comment:
There should also be a (new) `LuceneStandardTokenizerV2` that was added a couple weeks ago.